### DISCUSSION TITLE
Rotate/Resize/Move a window

### DISCUSSION THREAD

**Discussion by AngelosDid** [WindowTop/WindowTop-App#300] [Q&A]
Hello,

I have just bought the application but I cannot find a way to rotate/resize/move the windows selected by windowtop. Is there a guide for that?

**Comment by gileli121** [comment]
@AngelosDid 
Sorry for the late reply.
I usually look at emails and less at Discussions.

I don't understand what you mean by saying "rotate/resize/move".
Can you give examples of what you are trying to achieve?

Thanks
